src/docs/development/packages-and-plugins/using-packages.md

```diff
@@ -7,6 +7,30 @@ Flutter supports using shared packages contributed by other developers
 to the Flutter and Dart ecosystems. This allows quickly building
 an app without having to develop everything from scratch.
 
+{{site.alert.secondary}}
+  **What is the difference between a Flutter package
+  and a plugin?** Actually, a plugin is a _type_ of
+  package&mdash;the full designation is "plugin package",
+  which is generally shortened to "plugin".
+
+  **Packages**
+  : A package contains shared and reusable code and includes a list of
+    dependencies (specified in the `pubspec.yml` file), and (optionally)
+    Dart libraries, apps, resources, tests, images, and examples.
+    The [pub][] site lists many packages developed by Google engineers
+    and other generous members of the Flutter and Dart community
+    that you can use in your app.
```

Comment:
I found this a little hard to read, plus it seemed weird to imply that Google engineers are all generous. :) But maybe it should be "by Google engineers and other members..."

```suggestion
    The [pub.dev][] site lists many packages—developed by Google engineers
    and generous members of the Flutter and Dart community—
    that you can use in your app.
```

(and change [pub] to [pub.dev] below)